net-misc/gns3-gui/gns3-gui-2.0.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Graphical Network Simulator"
+HOMEPAGE="http://www.gns3.net/"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="test"
```

Comment:
Broken as in? Can't they be run manually providing proper running server and stuff?